### ISSUE TITLE
Regular menu icon (not open / close)

### ISSUE THREAD

**Issue by paulpooch** [palantir/blueprint#794]
There are open & close variants but not a regular 3 bar hamburger icon.

Not looking to start a holy war on hamburger menus.

**Comment by llorca** [comment]
What about `pt-icon-list`/`pt-icon-align-justify`? @pkwi what do you think?

**Comment by paulpooch** [comment]
I've resorted to unicode &#9776; for now.

Justify is four bars.  But I don't want a double hamburger menu  : )

**Comment by llorca** [comment]
@paulpooch Good news, `pt-icon-menu` is shipping next version: https://github.com/palantir/blueprint/pull/889

**Comment by paulpooch** [comment]
@llorca Awesome.  My next wishlist item is a 'send message' type icon along the lines of those paper airplane send message things.  You have an envelope but ideally something more like an arrow/paper airplane to show 'SEND' would be cool.

![image](https://cloud.githubusercontent.com/assets/539514/24422829/e354f30a-13c8-11e7-839c-76055a5c8b23.png)

**Comment by paulpooch** [comment]
My next wishlist item is a 'send message' type icon along the lines of those paper airplane send message things.  You have an envelope but ideally something more like an arrow/paper airplane to show 'SEND' would be cool.

![image](https://cloud.githubusercontent.com/assets/539514/24422829/e354f30a-13c8-11e7-839c-76055a5c8b23.png)

**Comment by llorca** [comment]
The good news was that the menu icon found a need with us. Unfortunately this isn't a way to request any icon you desire (though we'll enable customization of icon sets in 2.0!), nonetheless I'll keep you posted if this paper plane ever gets in

**Comment by paulpooch** [comment]
@llorca Yes I know.  Not an official request I know you have more urgent matters.

![image](https://cloud.githubusercontent.com/assets/539514/24478820/5bc08160-14aa-11e7-8514-15a6f0eac545.png)
